pkg/etcd/etcdkey.go

```diff
@@ -45,6 +45,9 @@ const (
 	DefaultClusterAndNamespacePrefix = "/tidb/cdc/default/default"
 	// DefaultClusterAndMetaPrefix is the default prefix of cluster meta
 	DefaultClusterAndMetaPrefix = "/tidb/cdc/default" + metaPrefix
+
+	// MigrateKeyPrefix is the prefix of backup keys during a migration
+	MigrateBackupPrefix = "/tidb/cdc/__backup__"
```

Comment:
Why is `BaseKey` not used here? It is better to construct the prefix using `BaseKey` so that we do not need to write `/tidb/cdc` over and over again.